lib/fluent/plugin/out_webhdfs.rb

```diff
@@ -66,6 +66,10 @@ class Fluent::Plugin::WebHDFSOutput < Fluent::Plugin::Output
   config_param :kerberos, :bool, default: false
   desc 'kerberos keytab file'
   config_param :kerberos_keytab, :string, default: nil
+  desc 'Use delegation token while upload webhdfs or not'
+  config_param :renew_kerberos_delegation_token, :bool, default: false
+  desc 'delegation token reuse timer (default 8h)'
+  config_param :renew_kerberos_delegation_token_time, :time, default: 8 * 60 * 60
```

Comment:
Sorry, one more thing.
Now I think `_interval` suffix is more appropriate for it than `_time`.
How do you think about it?